app/models/item.rb

```diff
@@ -6,4 +6,8 @@ class Item < ApplicationRecord
   has_many :orders, through: :order_items
 
   enum item_status: ['available', 'retired']
+
+  def get_quantity_in_order(order)
```

Comment:
Method looks good, do we want to maybe rename it? I'm not sure what this method does by looking at the name - is it the total quantity of order items? @meyerhoferc @CjMoore @valerietrudell